src/api/paths.js

```diff
@@ -0,0 +1,5 @@
+const apiUrl = {
+  USERS: "/users",
+}
+
+export default apiUrl;
```

Comment:
if you're exporting by default there is good practice to have file name equal export name

Reply:
Good point - I've changed it.